packages/ingest/http.js

```diff
@@ -136,13 +137,24 @@ module.exports.httpMixin = (superclass) => class extends superclass {
     const s3uri = buildS3Uri(bucket, key);
     log.info(`Sync ${remoteUrl} to ${s3uri}`);
 
+    let headers = {};
+    try {
+      const headResponse = await got.head(remoteUrl);
+      headers = headResponse.headers;
+    }
+    catch (err) {
+      log.info(`HEAD failed for ${remoteUrl} with error: ${err}.`);
+    }
+    const contentType = headers['content-type'] || mime.lookup(key);
```

Comment:
Grab the `mime-type` using an http `head` call. If something goes awry, try to infer the mime-type from `key`.